docs/1.0. Certificate Provisioning.md

```diff
@@ -147,58 +159,58 @@ The EST Server MUST present a valid TLS Server Certificate, this Certificate MUS
+The EST Server SHOULD support the server-side key generation endpoint `/serverkeygen` in accordance with [RFC 7030 - Section 4.4](https://tools.ietf.org/html/rfc7030#section-4.4). This endpoint allows devices unable to generate their own key pair with the required entropy to request a TLS certificate.
 
-If the server side generated key endpoint is supported, the EST Server MUST perform client authentication and authorisation as per [EST Server - Client Authentication](#est-server---client-authentication).
+If the server-side generated key endpoint is supported, the EST Server MUST perform client authentication and authorisation as per [EST Server - Client Authentication](#est-server---client-authentication).
 
 If the EST Client is authenticated and authorised, the EST Server MUST return the TLS Certificate and private key used to sign the certificate.
 
 If requested by the EST Client, the EST Server MUST encrypt the returned private key.
 
 The EST Server SHOULD generate a key of an appropriate length.
 
-It is RECOMMENDED that the EST Server signs the TLS Certificate using an RSA key, as currently they are the most computationally complex to generate.
+In order to accommodate low-powered devices it is RECOMMENDED that signing of TLS Certificates using an RSA key is done by the EST Server, as this is currently computationally complex.
 
-## EST Client
+## EST Client Behaviour
 
-The EST Client manufacturer SHOULD issue a unique TLS Client Certificate for every device. It is RECOMMENDED that the notAfter value be set to the GeneralizedTime value of 99991231235959Z, defined in [RFC 5280 - Section 4.1.2](https://tools.ietf.org/html/rfc5280#section-4.1.2.5) for when a certificate is intended to be used for the entire lifetime of the device. All certificates in the chain of trust MUST also have their NotAfter value set to the GeneralizedTime value of 99991231235959Z. A client certificate and chain of trust with the notAfter value set to the GeneralizedTime value of 99991231235959Z, MUST not be used for anything other than Client authentication.
+The EST Client manufacturer SHOULD issue a unique TLS Client Certificate for every device. It is RECOMMENDED that the `notAfter` value be set to the `GeneralizedTime` value of `99991231235959Z`, defined in [RFC 5280 - Section 4.1.2](https://tools.ietf.org/html/rfc5280#section-4.1.2.5) for when a certificate is intended to be used for the entire lifetime of the device. All certificates in the chain of trust MUST also have their `notAfter` value set to the `GeneralizedTime` value of `99991231235959Z`. A client certificate and chain of trust with the `notAfter` value set to the `GeneralizedTime` value of `99991231235959Z` MUST not be used for anything other than EST Client authentication.
```

Comment:
This section needs re-wording to match the changes in https://github.com/AMWA-TV/nmos-certificate-provisioning/pull/13 where the recommendation is to use a minimum of 30 years. 
Or should this be removed as it currently duplicates what is cover in the `EST Server - Client Authentication` section

Reply:
Thanks, James. Done.